src/main/java/org/sagebionetworks/web/client/widget/upload/MultipartUploaderImplV2.java

```diff
@@ -76,7 +90,7 @@ public void uploadFile(
       auth.getCurrentUserAccessToken(),
       fileName,
       blob,
-      storageLocationId.intValue(),
+      storageLocationIntValue,
```

Comment:
When the bug occurred, `storageLocationId` was null, so `storageLocationId.intValue()` yielded a NPE.

The SRC code that is called is not set up to accept `null`, so we have to supply the default storage location id here